grpc-google-cloud-bigquerystorage-v1beta1/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>6001</differenceType>
+    <className>com/google/cloud/bigquery/storage/v1beta1/BigQueryStorageGrpc</className>
+    <field>*</field>
```

Comment:
This ignores all field changes. That might be too broad.

Reply:
will be narrowing it down